plugin/Gui/AdminPageApiSettings.php

```diff
@@ -94,27 +91,40 @@ private function addFormModelAPI()
 		$pInputModelApiSecret = new InputModelOption('onoffice-settings', 'apisecret', $labelSecret, 'string');
 		$pInputModelApiSecret->setIsPassword(true);
 		$optionNameSecret = $pInputModelApiSecret->getIdentifier();
+		$apiSecret = get_option($optionNameSecret);
+		if (defined('ONOFFICE_CREDENTIALS_ENC_KEY')) {
+			try {
+				$apiSecretDecrypt = $this->_encrypter->decrypt(get_option($optionNameSecret), ONOFFICE_CREDENTIALS_ENC_KEY);
+			} catch (\RuntimeException $e) {
+				$apiSecretDecrypt = $apiSecret;
```

Comment:
This can be simplified

Reply:
i changed it, i add option ('ONOFFICE_CREDENTIALS_ENC_KEY') in database when encrypt apiSecret and i get option ('ONOFFICE_CREDENTIALS_ENC_KEY') when decrypt apiSecret, you review PR and suggest comment to finish this task .